qtpy/__init__.py

```diff
@@ -62,7 +62,11 @@
 
 """
 
-from distutils.version import LooseVersion
+try:  # disutils is being deprecated
+    import pkg_resources
```

Comment:
this import is useless

Reply:
I kept a backup plan, as I'm not sure on what very old Python QtPy may be used.

Reply:
ok, but it still useless